sdk/servicebus/Azure.Messaging.ServiceBus/tests/Samples/Sample10_ClaimCheck.cs

```diff
@@ -0,0 +1,71 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using NUnit.Framework;
+
+namespace Azure.Messaging.ServiceBus.Tests.Samples
+{
+    public class Sample10_ClaimCheck : ServiceBusLiveTestBase
+    {
+        [Test]
+        public async Task ClaimCheck()
```

Comment:
You're not cleaning up the data that this test writes.   That's going to have cost implications, especially for those that run the live tests locally and don't ever clean up their test resource deployment manually.

Reply:
If you decide to introduce the `StorageScope` that I mentioned in the test environment comments,  that would handle cleanup for us.